lms/envs/common.py

```diff
@@ -465,7 +465,16 @@
     # Enable organizational email opt-in
     'ENABLE_MKTG_EMAIL_OPT_IN': False,
 
-    # Show the mobile app links in the footer
+    # .. toggle_name: FEATURES['ENABLE_FOOTER_MOBILE_APP_LINKS']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Set to True if you want show the mobile app links (Apple App Store & Google Play Store) in
+    #   the footer.
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2015-01-13
+    # .. toggle_warnings: If you set this to True then you should also set your mobile application's app store and play
+    #   store URLs in the MOBILE_STORE_URLS settings dictionary.
```

Comment:
@jramnai It's probably worth mentioning that these links are not part of the default theme. It might be confusing for new operators if they enable this setting and nothing happens with the default theme.

There is an example of adding mobile links in the `https://github.com/edx/edx-platform/blob/master/themes/edx.org/lms/templates/footer.html`.

Reply:
A good catch @BbrSofiane. I totally missed that the `footer.html` file was under `edx.org` theme. Thanks!